smoke-test/tests/cypress/cypress/support/commands.js

```diff
@@ -143,6 +149,12 @@ Cypress.Commands.add('addTermToDataset', (urn, dataset_name, term) => {
   cy.contains(term);
 });
 
+Cypress.Commands.add("removeDomainFromDataset", (urn, dataset_name, domain_urn) => {
+  cy.goToDataset(urn, dataset_name);
+  cy.get('[class^="EntitySidebar__ContentContainer"] [href="/domain/' + domain_urn + '"] .anticon-close').click();
```

Comment:
This is not the ideal way to reference these HTML objects. Typically we use data-testid="custom-tag-id" to identify components. This way is prone to breaking with small refactoring, which can end up increasing ship-time